public/assets/scss/player/_player.scss

```diff
@@ -174,6 +174,15 @@
 			margin: 0;
 			cursor: pointer;
 		}
+
+		.error-el{
+			top: 50%;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered background, margin-left, margin-top, top, width